plugins/node.d/snmp__df_ram

```diff
@@ -178,12 +180,26 @@ if (defined $ARGV[0] and $ARGV[0] eq "config")
 
 	foreach my $part (keys %partitions)
 	{
-		print (&get_name_by_mp ($part), ".label ");
-		print (length($part)<=$MAXLABEL ? $part : "...".substr($part,-($MAXLABEL-3)));
-		print ("\n");
-		print (&get_name_by_mp ($part), ".warning 92\n");
-		print (&get_name_by_mp ($part), ".critical 98\n");
-		print (&get_name_by_mp ($part), ".info Usage for ". ($partitions{$part}{extinfo}||$part)."\n");
+                my $unit = &get_name_by_mp ($part);
+                print ($unit, ".label ");
+                print (length($part)<=$MAXLABEL ? $part : "...".substr($part,-($MAXLABEL-3)));
+                print ("\n");
+                if (defined $ENV{join('', $unit, "_warning")})
```

Comment:
I think, I am not an obsessively terse code writer, but maybe the following could be comparably readable, but less confusing? (at least for my reading habits)

```suggestion
                print($unit, ".warning ",  defined($ENV{"${unit}_warning"}) ? $ENV{"${unit}_warning"} : $warnlevel);
                print($unit, ".critical ",  defined($ENV{"${unit}_critical"}) ? $ENV{"${unit}_critical"} : $critlevel);
```

(at least I would like to get rid of the `join`. Additionally I enjoyed to remove he duplicate `print`, too.
What do you think?

Reply:
Ping?